src/components/UserForm/UserForm.js

```diff
@@ -31,92 +36,141 @@ class UserForm extends React.Component {
-                    {/* <div className="container__item">
-                        <label>PhotoUrl</label>
-                        <input
-                            value={this.state.email}
-                            type="text"
-                            onChange={this.photoUrlHandler}
-                        />
-                    </div> */}
-                    <div className="container__item">
-                        <button onClick={this.clickHandler}>Add</button>
-                    </div>
-                    <div>
-                        {/* <img src={"https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350"} /> */}
-                        {/* <img src = {this.props.url} alt='something'/> */}
-                    </div>
-                </form>
+            <>
+                <div className="box">
+                    <form className="userform">
+                        <div>
+                            <label className="label">Name</label>
+                            <input
+                                value={this.state.name}
+                                type="text"
+                                onChange={this.nameHandler}
+                            />
+                        </div>
+                        <div>
+                            <label className="label">Surname</label>
+                            <input
+                                value={this.state.surname}
+                                type="text"
+                                onChange={this.surnameHandler}
+                            />
+                        </div>
+                        <div>
+                            <label className="label">phone</label>
+                            <input
+                                pattern="[0-9]{3}-[0-9]{2}-[0-9]{2}-[0-9]{2}"
+                                value={this.state.phone}
+                                type="tel"
+                                id="phone"
+                                onChange={this.phoneHandler}
+                            />
+                        </div>
+                        <div>
+                            <label className="label">email</label>
+                            <input
+                                value={this.state.email}
+                                type="text"
+                                onChange={this.emailHandler}
+                            />
+                        </div>
+                        <div>
+                            <label className="label">PhotoUrl</label>
+                            <input
+                                value={this.state.photo}
+                                type="text"
+                                onChange={this.photoHandler}
+                            />
+                        </div>
+                        <div>
+                            <button className="btn" onClick={this.clickHandler}>
```

Comment:
for the form don't pass onClick to the form's button, your click handler pass to form's onSubmit.